2023/Greatman-Ugorji/README.md

```diff
@@ -56,17 +56,35 @@ To stand out as an applicant,
+- [x] Write a short blog post which demonstrates your understanding of **accessibility**. In English, with at least one image, less than 500 words. Demonstrate your understanding _based on past experiences_. For example, any specific accessibility improvements you’ve done in the past, or specific issues you identified. The more specific the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
+- [x] Write a short blog post which demonstrates your expertise in **technical writing**. In English, with at least one image, less than 500 words. Demonstrate your expertise _based on past experiences_. For example, any specific documentation writing you’ve done in the past, or other technical writing you’ve made. The more specific the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
 - [ ] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to a Wagtail website. Once you’ve received access, we ask you to create a new page demonstrating some of Wagtail’s accessibility features.
 - [ ] With your access to the demo site and by reviewing our [user guide](https://guide.wagtail.org/), identify a feature of Wagtail which isn’t documented, and report it back on the [guide project issue tracker](https://github.com/wagtail/guide/issues). Make sure to explain what the feature is, why and how you think it should be documented.
 - [ ] Draft the new content to document the feature of Wagtail which you’ve reported isn’t documented. Add it to the issue you created as a new comment, so we can provide feedback.
 - [ ] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to [guide.wagtail.org](https://guide.wagtail.org/). Once you’ve received access, we ask you to make the necessary changes in the CMS to add the feature documentation.
 - [ ] TBC – Additional tasks specific to the Outreachy application process.
+
+## Research
+
+[Wagtail User Guide](https://guide.wagtail.org/en-latest/)
+[Wagtail Documentation](https://docs.wagtail.org/en/stable/)
+[Caktus Group](https://www.caktusgroup.com/blog/2019/03/07/why-we-love-wagtail/)
+
+### Short Blog Post - Recent Learning
+
+[My Recent Learning](https://0xryker.hashnode.dev/beginners-guide-to-cryptocurrency?source=more_articles_bottom_blogs)
+
+### Accessibility Post
+
+[Accessibility Post](https://0xryker.hashnode.dev/a-basic-introduction-to-web-accessibility)
```

Comment:
This is a bit off topic :/ I don’t see anything on there that demonstrates your expertise based on past experience.

> As you can see, web accessibility makes more people use a website. This also leads to increased revenue for the website owners.

To be honest though both of those things are good, they’re not the main point of accessibility. I think you should revisit the fundamental reasons why it matters.